ambry-api/src/test/java/com.github.ambry/account/InMemAccountService.java

```diff
@@ -114,9 +114,10 @@ public synchronized void clear() {
    * Creates and adds an {@link Account} to this {@link AccountService}. The account will contain one container
    * with {@link Container#DEFAULT_PUBLIC_CONTAINER_ID}, one with {@link Container#DEFAULT_PRIVATE_CONTAINER_ID} and
    * one other random {@link Container}.
+   * @param enableEncryption {@code true} if encryption needs to be enabled. {@code false} otherwise
    * @return the {@link Account} that was created and added.
    */
-  public synchronized Account createAndAddRandomAccount() {
+  public synchronized Account createAndAddRandomAccount(boolean enableEncryption) {
```

Comment:
don't do this - this is supposed to be a method that takes no params and makes it easy to just create a random account. Add whatever containers you want after.